app/utils/kms_utils.py

```
@@ -0,0 +1,187 @@
+        logger.info("Starting AWS KMS decryption")
+        ciphertext = base64.b64decode(encrypted_string)
+        
+        response = _aws_kms_client.decrypt(CiphertextBlob=ciphertext)
+        
+        if not response.get('Plaintext'):
+            logger.error("No plaintext in AWS KMS response")
+            return None
+        
+        decrypted = response['Plaintext'].decode('utf-8')
+        logger.info("AWS KMS decryption successful")
+        return decrypted
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)